localization/english/wc_buildings_l_english.yml

```
@@ -0,0 +1,7 @@
+﻿l_english:
+ building_type_dark_portal_01:0 "Dark Portal"
+ building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds: Azeroth and Draenor, allowing alien creatures to go back and forth."
+ building_dark_portal_01:1 "Dark Rift"
+ building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds."
+ building_dark_portal_02:1 "Dark Portal"
+ building_dark_portal_02_desc:1 "A massive arch with magical runes that maintain the connection between the two worlds allowing large armies to pass the portal."
```

added a comma and a word.
```suggestion
 building_dark_portal_02_desc:1 "A massive arch with magical runes that maintain the connection between the two worlds, allowing large armies to pass through the portal."
```